tests/test-pipenv.bsh

```diff
@@ -18,3 +19,39 @@ begin_test "Pipenv"
 
 )
 end_test
+
+# test 30_get-pipenv script without a container
+# Note this test assumes a system-wide python is available
+begin_test "30_get-pipenv"
+(
+  setup_test
+
+  # environment
+  export PIPENV_VERSION="2020.8.13"
+  export LC_ALL=C.UTF-8
+  export LANG=C.UTF-8
+
+  # output directory
+  OUTPUT_DIR="$(mktemp -d)"
```

Comment:
We write everything in a test to somewhere in "${TESTDIR}". Please change to use something in there.

Examples:

1. `OUTPUT_DIR="${TESTDIR}/whatever"` That's what we normally do
2. `mktemp -d -p "${TESTDIR}"` isn't going to work out out, as `-p` won't work on macOS. There is a "compatible" version being added [here](https://github.com/VisionSystemsInc/vsi_common/pull/268/files#diff-612f849c52ef46c5a2276d3283402618R156-R177).

Reply:
Fixed - since the `30_get-pipenv` script also does a `mktemp`, we use the `export TMPDIR="${TESTDIR}"` trick.